src/Button/Button.js

```diff
@@ -56,9 +56,9 @@ export const Button = ({ className, children, onClick, ref, variant, size, type,
   // Is disabled
   if (isDisabled) {
     levers.disabled = classnames('su-bg-black-20 su-text-black su-border-2 su-border-black-20 su-border-solid su-pointer-events-none')
+    levers.variant = classnames(levers.variant, 'su-border-none', {'su-bg-digital-red': false, 'su-border-solid': false, 'su-text-digital-red': false, 'su-text-white': false})
```

Comment:
I'm going to merge this in, but will likely tweak this since we're keeping the border on all the buttons (I don't like the way the 2nd-dary button uses the inner shadow to create the borders since it's really hard to change shadow colors/width in TW. So all buttons have borders, but for primary/disabled they're the same color as the button backgrounds.